spec/lib/api/v3/queries/schemas/query_schema_representer_spec.rb

```diff
@@ -357,7 +357,11 @@
              Queries::WorkPackages::Columns::PropertyColumn.new(:bogus2),
              Queries::WorkPackages::Columns::PropertyColumn.new(:bogus3),
              Queries::WorkPackages::Columns::RelationToTypeColumn.new(type),
-             Queries::WorkPackages::Columns::RelationOfTypeColumn.new(name: :label_relates_to, sym: :relation1)]
+             Queries::WorkPackages::Columns::RelationOfTypeColumn.new(
+               name: :label_relates_to,
+               sym: :relation1,
+               sym_name: :label_relates_to
+             )]
```

Comment:
This was quite a bit of detective work! Nice

Reply:
Yep, thanks for helping in the process!🙌